### ISSUE TITLE
Static Local Authority Lookup

### ISSUE THREAD

**Issue by no-reply** [curationexperts/chf-sufia#16]
Depends on #15

**Comment by no-reply** [comment]
We could consider simply using the YAML versions that ship with QA: https://github.com/projecthydra-labs/questioning_authority/#local-sub-authorities

It should be possible to use URIs as ids in this context, and we could probably use the `QA::LDF::Model` code seamlessly.

**Comment by no-reply** [comment]
cc: @hackmastera Do you have thoughts about a very lightweight version of this that simply adds URI storage to the current local authorities code?

**Comment by hackartisan** [comment]
@no-reply Yes the YAML configs that ship with QA were what I had initially envisioned; that seems fine!
